### PR TITLE
[android] Improve languages config 

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**
This PR follow works started here: #3619
I have try to rebase branch but with merge branch, i don't have find how push commit without send all changes from merge branch

### CODE REVIEW

**Comment by biodranik** [comment]
Please do not close PRs and create new ones in the future. We can help to clean everything up without recreating. And with a proper rebasing it is not necessary.

File: tools/unix/generate_localizations.sh [hunk 1]
```diff
@@ -66,4 +66,10 @@ cat "$STRINGS_PATH"/{strings,types_strings}.txt> "$MERGED_FILE"
 "$TWINE" generate-all-localization-files --format jquery "$OMIM_PATH/data/countries_names.txt" "$OMIM_PATH/data/countries-strings/"
 "$TWINE" generate-all-localization-files --format jquery "$OMIM_PATH/data/strings/sound.txt" "$OMIM_PATH/data/sound-strings/"
 
+# Generate list of languages and add list in gradle.properties to be used in build.gradle in resConfig
+SUPPORTED_LOCALIZATIONS="supportedLocalizations="$(sed -nEe "s/ +([a-zA-Z]{2}(-[a-zA-Z]{2,})?) = .*$/\1/p" "data/strings/strings.txt" | sort -u | tr '\n' ',' | sed -e 's/-/_/g' -e 's/,$//')
+if [ "$SUPPORTED_LOCALIZATIONS" != "$(grep listLanguages "$OMIM_PATH/android/gradle.properties")" ]; then
+	sed -i 's/supportedLocalizations.*/'"${SUPPORTED_LOCALIZATIONS}"'/' "$OMIM_PATH/android/gradle.properties"
```
[the remark this answers is not in the document]
Comment (Jean-BaptisteC) [reply tools/unix/generate_localizations.sh]: Done

File: android/gradle.properties [hunk 2]
```diff
@@ -8,3 +8,5 @@ org.gradle.jvmargs=-Xmx1024m -Xms256m
 android.useAndroidX=true
 android.enableJetifier=true
 android.native.buildOutput=verbose
+# Autogenerated by tools/unix/generate_localizations.sh
+supportedLocalizations=ar,be,bg,ca,cs,da,de,el,en,en_GB,es,es_MX,et,eu,fa,fi,fr,fr_CA,he,hi,hu,id,in,it,ja,ko,mr,nb,ne,nl,pl,pt,pt_BR,ro,ru,sk,sv,sw,th,tr,uk,vi,zh_Hans,zh_Hant
```
Comment (rtsisyk) [inline android/gradle.properties:12]: nit: missing new line

**Review by rtsisyk — APPROVED** [review]

File: android/gradle.properties (hunk 2 above)
Comment (Jean-BaptisteC) [reply android/gradle.properties]: Done

**Review by biodranik — APPROVED** [review]
Congrats! You've done it )

**Pull request merged**